### PR TITLE
Arcadia drip

### PR DESCRIPTION
**Pull request by AjexRose**
## About the PR

Added clothing for the faction of Arcadia Industries as well as a vendor for them
The vendor is temporarily available as a Expedition reward, as a way for players to unlock it until a PoI for Arcadia Industries is made. 

## Why / Balance

During the townhall, there were mentions of inserting more player made factions into the frontier, so I thought this would be a good start. 

**Changelog**

added: Arcadia Industries is now beginning marketing in the frontier with unique clothing items, unlockable by those with combat prowess in expeditions

### CODE REVIEW

**Comment by Cheackraze** [comment]
love it.  My only issue is, we need to keep all of our entities separated from the base game files. So for all of your yml entries, we need new files created inside /prototypes/_NF/ instead of adding to the vanilla ones.

This is so that maintainers (just me for now) have a much, *MUCH* easier time when it comes to bringing in changes from vanilla SS14 so that the files do not conflict.  Once we get everything split into their own files, we can get this in and start making a POI for you guys!

**Comment by AjexRose** [comment]
Alrighty, I'll try to get all the files to the _NF section of the code!

**Comment by AjexRose** [comment]
@Cheackraze The Arcadia Clothing hopefully all fully within the _NF folders now.

File: Resources/Prototypes/_NF/Entities/Clothing/Belt/belts.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  id: ClothingBeltArcadia
+  name: arcadia webbing
+  description: A webbing created by Arcadia Industries. Seems very capable of fitting many items.
+  components:
+- type: Sprite
+  sprite: _NF/Clothing/Belt/arcadia-webbing.rsi
+- type: Clothing
+  sprite: _NF/Clothing/Belt/arcadia-webbing.rsi
+- type: Storage
+  capacity: 60
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Entities/Clothing/Belt/belts.yml:12]: yml is sensitive to indentation, so everything under the `  components:` needs to get formatted with 1 more double space indent

**Review by Cheackraze — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_NF/Entities/Clothing/Masks/masks.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+- type: entity
+  parent: ClothingMaskBase
+  id: ClothingMaskArcadia
+  name: arcadia battle mask
+  description: A close-fitting high tech mask designed by Arcadia Industries for space faring battlers.
+  components:
+  - type: Sprite
+    sprite: _NF/Clothing/Mask/arcadia-battleface.rsi
+  - type: Clothing
+    sprite: _NF/Clothing/Mask/arcadia-battleface.rsi
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Entities/Clothing/Masks/masks.yml:10]: typo. should be /Masks/ ill fix it so we can merge it

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**